elasticdl/python/ps/parameter_server.py

```diff
@@ -29,6 +30,9 @@ def __init__(self, args):
         # Create Parameters instance
         self.parameters = Parameters()
 
+        self.master_name = "elasticdl-%s-master" % args.job_name
```

Comment:
Can we put the naming format in a common file. I think multiple component will use this.

Reply:
You can reuse `k8s_client.Client.get_master_pod_name()`

Reply:
Yes, but only master has a k8s_client object. I will make master_name as an argument, and pass to PS